Sources/Defaults/Defaults+AnySerializable.swift

```diff
@@ -0,0 +1,257 @@
+import CoreGraphics
+import Foundation
+
+public protocol DefaultsAnySerializableProtocol: Defaults.Serializable {
+	init<Value: Defaults.Serializable>(_ value: Value)
+	func get<Value: Defaults.Serializable>() -> Value?
+	func get<Value: Defaults.Serializable>(_: Value.Type) -> Value?
+}
+
+extension Defaults {
+	public typealias AnySerializableProtocol = DefaultsAnySerializableProtocol
+	/**
+	Have an internal property `value` which` should always be a UserDefaults native supported type.
```

Comment:
The first sentence of a type should describe what it is, and then what problem it solves.